src/db/spanner/models.rs

```diff
@@ -1270,20 +1273,23 @@ impl SpannerDb {
             sqlparams.insert("newer".to_string(), newer.as_rfc3339()?.to_spanner_value());
             sqlparam_types.insert("newer".to_string(), as_type(TypeCode::TIMESTAMP));
         }
-        query = match sort {
-            // issue559: Revert to previous sorting
-            /*
-            Sorting::Index => format!("{} ORDER BY sortindex DESC, bso_id DESC", query),
-            Sorting::Newest | Sorting::None => {
-                format!("{} ORDER BY modified DESC, bso_id DESC", query)
-            }
-            Sorting::Oldest => format!("{} ORDER BY modified ASC, bso_id ASC", query),
-            */
-            Sorting::Index => format!("{} ORDER BY sortindex DESC", query),
-            Sorting::Newest => format!("{} ORDER BY modified DESC", query),
-            Sorting::Oldest => format!("{} ORDER BY modified ASC", query),
-            _ => query,
-        };
+
+        if self.stabilize_bsos_sort_order() {
```

Comment:
Sorry Ethan, I probably wasn't clear enough about this: let's solely re-enable _only this_ section -- `Sorting::None` now acts like `Newest` and enabling `bso_id` as a secondary sort.

The rest of the related bits for issue559 should stay commented out. They were related to enabling this sorting but are really a different feature, they enable "encoded offsets" -- a different format of offset that potentially optimized the get queries during pagination.

The sorting changes were required to enable the encoded offset but we don't need (or want) the encoded offsets for solely this sorting change. `encode_next_offset`, the `extractor` changes and the related "timestamp always None" change above in this file should stay the same (commented out).

Reply:
Ah, my bad! I misunderstood. For some reason, I assumed the secondary sort required the other offset change. I'll go ahead and fix it